server/api/schemes/userSchema.js

```diff
@@ -14,6 +14,8 @@ const userSchema = mongoose.Schema({
     getToKnowState: { type: String, require: false, default: "uninitialized" },
     online: { type: Boolean, require: false, default: undefined },
     confirm: { type: Boolean, require: true, default: false },
+    aboutMe: {type: String, require: false, default: ""},
```

Comment:
Default is "" in the place you search for available psychologists you search for a users that have phone and about me. In this case the condition is always true. So please change default to undefined